vispy/scene/widgets/widget.py

```diff
@@ -224,21 +244,21 @@ def add_grid(self, *args, **kwargs):
         """
         Create a new Grid and add it as a child widget.
 
-        All arguments are given to add_widget().
+        All arguments are given to Grid().
```

Comment:
I'm not sure this is a problem, `Grid` has its own signature for both `add_grid` and `add_widget`, and they don't call the super `Widget` versions...?